src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -71,16 +75,27 @@ def invalid_metric_tags(tags: Mapping[str, str]) -> Sequence[str]:
     return invalid_strs
 
 
+def extract_use_case_id(mri: str) -> Optional[UseCaseId]:
+    """
+    Returns the use case ID given the MRI, returns None if MRI is invalid.
+    """
+    use_case_mapping = {"transactions": "performance", "sessions": "release-health"}
```

Comment:
Now that `UseCaseID` is defined and we have transactions/sessions instead or performance/release health, let's remove this extra mapping